data/presets/highway/path/traffic_island_shared.json

```diff
@@ -0,0 +1,31 @@
+{
+    "locationSet": {
+        "include": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    },
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "path": "traffic_island"
+    },
+    "addTags": {
+        "highway": "path",
+        "path": "traffic_island"
+    },
```

Comment:
Same as above, IMO

```suggestion
    "removeTags": {
        "crossing:island": "*",
    },
```

Reply:
I'm a bit confused here. Is `crossing:island` meant to be used on ways with `*=traffic_island`? [The wiki page](https://osm.wiki/Key:crossing:island)doesn't mention this. 

There are also [344 ways](https://overpass-turbo.eu/s/1VZg) with `*=traffic_island`+`crossing:island=no`, which seems like a contradiction. Most that I checked were the result of crossing ways being split, and the crossing tags being accidentally carried over to the island.

Reply:
lets discuss this one in [the other thread](#discussion_r1894376539)